### PR TITLE
Kristen- added public art piece detail 3

### PR DESCRIPTION
**Pull request by KristenOmori-NOAA**
Added the The Seas are Rising and So are We art work under detail 3.

### CODE REVIEW

File: public_art.md [hunk 1]
```diff
@@ -10,6 +10,16 @@ Add details here including images, hours, location, etc.
 ### Detail 2
 Add details here including images, hours, location, etc.
 
+### Detail 3
+![image](https://user-images.githubusercontent.com/132607494/236925590-f705ceb1-cabc-468d-b724-509e8651e3a7.png)
+
+**The Seas are Rising and So are We** <br>
+ARTIST: Josie Morway <br>
```
Comment (SteveTeo-NOAA) [inline public_art.md:17]: Rejected!!!!

**Review by SteveTeo-NOAA — CHANGES_REQUESTED** [review]
Changed the year

File: public_art.md (hunk 1 above)
Comment (KristenOmori-NOAA) [reply public_art.md:17]: FINE

**Review by KristenOmori-NOAA — COMMENTED** [review]
incorrect

**Pull request merged**